src/services/WebhookService.ts

```diff
@@ -8,62 +8,21 @@ const fetchShopifyWebhooks = async (payload?:  any): Promise <any>  => {
   });
 }
 
-// TODO: add the service endpoint for the new order webhook
-const subscribeNewOrderWebhook = async (payload?: any): Promise <any> => {
-  return api ({
-    url: '',
-    method: 'post',
-    data: payload
-  })
-}
-
-// TODO: add the service endpoint for the cancelled order webhook
-const subscribeCancelledOrderWebhook = async (payload?: any): Promise <any> => {
-  return api ({
-    url: '',
-    method: 'post',
-    data: payload
-  })
-}
-
-// TODO: add the service endpoint for the payment status webhook
-const subscribePaymentStatusWebhook = async (payload?: any): Promise <any> => {
-  return api ({
-    url: '',
-    method: 'post',
-    data: payload
-  })
-}
-
-// TODO: add the service endpoint for the order return webhook
-const subscribeReturnWebhook = async (payload?: any): Promise <any> => {
-  return api ({
-    url: '',
-    method: 'post',
-    data: payload
-  })
-}
-
-// TODO: add the service endpoint for the new product webhook
-const subscribeNewProductsWebhook = async (payload?: any): Promise <any> => {
-  return api ({
-    url: '',
-    method: 'post',
-    data: payload
-  })
-}
-
-const subscribeDeleteProductsWebhook = async (payload?: any): Promise <any> => {
-  return api ({
-    url: 'service/subscribeProductDeleteWebhook',
-    method: 'post',
-    data: payload
-  })
-}
+// TODO: add the service endpoint for the new order webhook, cancelled order webhook, payment status webhook, order return webhook & new product webhook.
+const webhookMethods = {
+  'NEW_ORDERS': '',
+  'CANCELLED_ORDERS': '',
+  'PAYMENT_STATUS': '',
+  'RETURNS': '',
+  'NEW_PRODUCTS': '',
+  'DELETE_PRODUCTS': 'service/subscribeProductDeleteWebhook',
+  'BULK_OPERATIONS_FINISH': 'service/subscribeFileStatusUpdateWebhook'
+} as any
 
-const subscribeFileStatusUpdateWebhook = async (payload?: any): Promise <any> => {
+const subscribeWebhook = async (payload?: any, id?: string): Promise <any> => {
+  const endpointUrl = webhookMethods[id as string];
```

Comment:
Handle case when endpoint is not available like this
```
    if (!webhookMethod) {
      showToast(translate("Configuration missing"));
      return;
    }
 ```

Reply:
Done sir!

Reply:
I think we should improve the object name `webhookMethods`